.github/workflows/facades.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - master
       - '*.x'
+  workflow_dispatch:
```

Comment:
Adding `workflow_dispatch` allows us to manually trigger the action without creating new commit:

![CleanShot 2023-12-07 at 08 54 18](https://github.com/laravel/framework/assets/172966/107dfd6a-faf9-4cc1-a516-9cdef135f082)